### ISSUE TITLE
Sindragosa does not walk on the floor

### ISSUE THREAD

**Issue by tje3d** [TrinityCore/TrinityCore#21904]
**Description:**

Sindragosa does not walk on the floor

![2](https://user-images.githubusercontent.com/5238989/39626598-d8e32c9e-4fb7-11e8-85ea-4febc81ce986.png)
![1](https://user-images.githubusercontent.com/5238989/39626599-d9110b96-4fb7-11e8-94eb-a186f9bebc3f.png)

**Steps to reproduce the problem:**

1. .go xyz 4324 2484 211 631
2. Start combat

**Branch(es):** 3.3.5

**TC rev. hash/commit:** 081eab3cf5a5ec42917b2a7eaebf816fb36da423

**Operating system:** Ubuntu 17.10

**Comment by Foereaper** [comment]
https://github.com/TrinityCore/TrinityCore/issues/21872

See my comment here

**Comment by tje3d** [comment]
@Foereaper that works!
```diff
diff --git a/src/server/game/Movement/PathGenerator.cpp b/src/server/game/Movement/PathGenerator.cpp
index 03f383b2fd..e8239afd4e 100644
--- a/src/server/game/Movement/PathGenerator.cpp
+++ b/src/server/game/Movement/PathGenerator.cpp
@@ -582,7 +582,7 @@ void PathGenerator::NormalizePath()
 {
     for (uint32 i = 0; i < _pathPoints.size(); ++i)
     {
-        _pathPoints[i].z += _sourceUnit->GetCollisionHeight();
+        // _pathPoints[i].z += _sourceUnit->GetCollisionHeight();
         _sourceUnit->UpdateAllowedPositionZ(_pathPoints[i].x, _pathPoints[i].y, _pathPoints[i].z);
     }
 }
```

**Comment by tje3d** [comment]
Fixed by d9a2177e9cc1115b0d830bd0a76ac0eec9878e05
